decorator/trim_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.strip
+  end
```

Comment:
- [ ] Let's trim the word only if the name has a maximum of 10 character.
`Hint`: you should use [ternary operator](https://www.rubyguides.com/2019/10/ruby-ternary-operator/) to achieve this.